app/(root)/posts/page.tsx

```diff
@@ -4,16 +4,15 @@ import Link from 'next/link';
 const Posts = () => {
```

Comment:
A little confused about whether this component will be shared between the left side Navbar and the main area where the posts are displayed

Reply:
Faizan confirmed that these posts will be the same, just that the navbar organizes them according to 'post type' and 'most recent', and the main body of posts are organized just by 'most recent'.